gooddata-sdk/docs/services/catalog-data-source.rst

```diff
@@ -228,6 +228,22 @@ The *gooddata_sdk.catalog_data_source* supports the following action API calls:
 
     Invalidate cache of your computed reports to force your analytics to be recomputed.
 
+* ``scan_data_source(data_source_id: str, scan_request: CatalogScanModelRequest = CatalogScanModelRequest(), report_warnings: bool = False)``
+
+    Returns *CatalogScanResultPdm*.
+
+    Scan data source specified by its id and optionally by specified scan request. Additional parameter report_warnings can be passed to suppress or to report warnings. The parameter report_warnings is by default set to False.
```

Comment:
By default warnings are returned but not reported to STDOUT. If you set report_warnings to True, warnings are reported to STDOUT.